clang/include/clang/APINotes/Types.h

```diff
@@ -600,6 +603,15 @@ class FunctionInfo : public CommonEntityInfo {
 
   friend bool operator==(const FunctionInfo &, const FunctionInfo &);
 
+  FunctionInfo &operator|=(const FunctionInfo &RHS) {
```

Comment:
Are we using this function somewhere? I can't seem to find the usage.

Reply:
We can remove `|=` operator for `FunctionInfo`. I  just added it as other types had such operator. But it is probably not necessary for this patch!